src/lamplib/src/genny/tasks/mothra_service.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+import subprocess
+import tempfile
+from typing import Optional
+import structlog
+import yaml
+
+MOTHRA_REPO = "git@github.com:10gen/mothra.git"
```

Comment:
SSH repositories are going to stop working within evergreen soon (DEVPROD-351). We will need to get evergreen to provide us a short-lived SSH token. 

I am okay with this code getting merged in as-is so long as we have a plan for conversion and we carry it out soon (cc @lgfang who is handling this for my team).